relay-cardinality/src/redis/quota.rs

```diff
@@ -125,6 +126,34 @@ impl std::ops::Deref for QuotaScoping {
     }
 }
 
+// Required for hashbrown's `entry_ref`.
+impl From<&QuotaScoping> for QuotaScoping {
+    fn from(value: &QuotaScoping) -> Self {
+        value.clone()
+    }
+}
```

Comment:
Why not clone directly? Hiding this clone here may be harder to track if it becomes a problem in the future.

Reply:
See the comment above, this impl is required for `hashbrown`'s `entry_ref`. There is unfortunately no way around that if we want to use `entry_ref`.